### PR TITLE
pageable

### PR DESCRIPTION
**Pull request by RomanukAlex**
Add pageable. Correct modules dao, service and rest.

### CODE REVIEW

File: rest-app/src/main/java/com/epam/repair/rest/controller/StreetController.java [hunk 1]
```diff
@@ -3,11 +3,13 @@
 import com.epam.repair.model.Street;
 import com.epam.repair.service.StreetService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.List;
 
+@CrossOrigin
```
Comment (aleksieiev) [inline rest-app/src/main/java/com/epam/repair/rest/controller/StreetController.java:12]: What for?

File: rest-app/src/test/java/com/epam/repair/rest/StreetControllerIT.java [hunk 2]
```diff
@@ -45,4 +41,4 @@ public void findAll() throws Exception {
 
         JSONAssert.assertEquals(expected, actual, true);
     }
-}
+}
```
Comment (aleksieiev) [inline rest-app/src/test/java/com/epam/repair/rest/StreetControllerIT.java:44]: Very important changes ;-)

**Review by aleksieiev — APPROVED** [review]

**Comment by aleksieiev** [comment]
Keep in mind that there's a problems
Conflicting files
rest-app/src/test/java/com/epam/repair/rest/RepairRepairOrderControllerIT.java

File: rest-app/src/main/java/com/epam/repair/rest/controller/StreetController.java (hunk 1 above)
Comment (RomanukAlex) [reply rest-app/src/main/java/com/epam/repair/rest/controller/StreetController.java:12]: Web application can not work with JSON with secure cross-origin requests

**Review by sergeyarhutik — APPROVED** [review]

File: rest-app/src/main/java/com/epam/repair/rest/controller/StreetController.java (hunk 1 above)
Comment (sberdachuk-epam) [reply rest-app/src/main/java/com/epam/repair/rest/controller/StreetController.java:12]: Hi Alex, 
Method or controller CrossOrigin annotation is not flexible if you need a secure this application in a feature. Please review the possibility to create a global CORS configuration for the whole application in one place.

**Pull request merged**